### PR TITLE
fixes #273

### LINKED ISSUE #273: Add link to lfx insights site from the contributors page
Here is a link to the community statistics from Hyperledger https://insights.lfx.linuxfoundation.org/projects/hyperledger-f

### PR DESCRIPTION
**Pull request by Swastyy**
Signed-off-by: Swastika Gupta <gupta.swastika521@gmail.com>

### CODE REVIEW

**Comment by Swastyy** [comment]
Hey @arsulegai , here is the PR that passes the checks.

File: index.md [hunk 1]
```diff
@@ -45,3 +45,5 @@ possible by generous contributions of
 ## GitHub Action Contributors
 <ul class="list-style-none">  <li class="d-inline-block mr-1"> <a href="https://github.com/arsulegai"><img src="https://avatars.githubusercontent.com/u/27664223?v=4" width="32" height="32" alt="27664223"></a> </li>  <li class="d-inline-block mr-1"> <a href="https://github.com/hyperledger-bot"><img src="https://avatars.githubusercontent.com/u/76175814?v=4" width="32" height="32" alt="76175814"></a> </li>  <li class="d-inline-block mr-1"> <a href="https://github.com/nidhi-singh02"><img src="https://avatars.githubusercontent.com/u/38173192?v=4" width="32" height="32" alt="38173192"></a> </li>  <li class="d-inline-block mr-1"> <a href="https://github.com/ryjones"><img src="https://avatars.githubusercontent.com/u/466142?v=4" width="32" height="32" alt="466142"></a> </li>  </ul>
 
+## Github Community Statistics
+Over the time, we can see the increasing number of contributions in the [Community statistics](https://insights.lfx.linuxfoundation.org/projects/hyperledger-f/dashboard) from Hyperledger.
```
Comment (arsulegai) [inline index.md:49]: [nitpick] How about wording this paragraph this way?
```
Get the pulse across Hyperledger projects at [Community Statistics](https://insights.lfx.linuxfoundation.org/projects/hyperledger-f/dashboard) page of the LFX Insights portal.
```

Comment (Swastyy) [reply index.md]: Hi @arsulegai done the changes. Please review it now. Thanks in advance.

**Review by arsulegai — APPROVED** [review]

**Comment by arsulegai** [comment]
Thank you for the contribution.

**Pull request merged**